hal/common/retarget.cpp

```diff
@@ -529,10 +536,14 @@ extern "C" WEAK void mbed_sdk_init(void) {
 extern "C" int $Super$$main(void);
 
 extern "C" int $Sub$$main(void) {
-    mbed_sdk_init();
```

Comment:
Ok I changed the place where mbed_sdk_init is called in commit:
	INIT:GCC with rtos: Align pre-main initialization steps between TCs